core/services/relay/evm/mercury/v3/transmitter_test.go

```diff
@@ -0,0 +1,62 @@
+package mercury_v3
```

Comment:
Why does `transmitter_test` live in each v1, v2, v3 package etc? Should it not live next to `transmitter.go`?

Reply:
All common transmitter tests are still located next to the `transmitter.go`. This one is to test the `Transmit` func that requires building the specific report. Although, since any specific report is just a blob of bytes, we could test only one report version in the root `transmitter_test.go`